wdio.conf.js

```diff
@@ -24,8 +24,41 @@ const config = {
     enabled: !process.env.TRAVIS,
   },
 
+  maxInstances: 1,
```

Comment:
Isn't the default 1?

Reply:
i think toolkit had 3 configured for somereason. Once we cut a release it won't matter though

Reply:
this change probably isn't needed though